src/api/apiClient.ts

```diff
@@ -109,10 +109,18 @@ export const errorMap = (issue: ZodIssueOptionalMessage, ctx: ErrorMapCtx) => {
     }
 }
 
-export const apiClient = createApiClient(BASE_URL, {
+// TLDR: the inferred TS schema was too big, so this is a workaround to fix it.
```

Comment:
so what does this actually mean for the result/effect of this? mostly curious

Reply:
basically removes a typescript compiler warning where the infered type schema was too long, this basically sets the type to a defined type.

Reply:
so what does a type schema that's too long cause?

Reply:
ah got it